qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -181,6 +181,16 @@ def write_local_config(self):
         self.mount_a.client_remote.write_file(join(self.xfstests_repo_path, 'local.config'),
                                               xfstests_config_contents, sudo=True)
 
+    def write_ceph_exclude(self):
```

Comment:
something like `exclude_ceph_write_tests()` looks meaningful

Reply:
I just followed `write_local_config`. Let's just use `write_ceph_exclude` to make it to be consistent with the existing one.

Reply:
im fine with it but `write_local_config` makes sense cause that func is writing to local config, but when I read `write_ceph_exclude` then only after reading code I can come to know that it's actually a set of tests that you are trying to exclude. Also if you look at the function names above `write_local_config()`, they dont follow any pattern, they just explain what they do.

Reply:
Both are generating the config files for xfstests-dev test cases, not writing tests for them.

Reply:
alright